firebase-firestore/src/main/java/com/google/firebase/firestore/local/LocalStore.java

```diff
@@ -357,7 +361,7 @@ public SnapshotVersion getLastRemoteSnapshotVersion() {
                   oldQueryData
                       .withResumeToken(resumeToken, remoteEvent.getSnapshotVersion())
                       .withSequenceNumber(sequenceNumber);
-              targetIds.put(boxedTargetId, newQueryData);
+              queryDataByTarget.put(boxedTargetId, newQueryData);
```

Comment:
preexisting nit: this will have to un-box so you might as well pass `targetId`

Reply:
Fixed